### DISCUSSION TITLE
migrate

### DISCUSSION THREAD

**Discussion by forever218** [hexojs/hexo#4956] [Question / Answer] [answered]
How to migrate to a new computer?🤔Copying the folder directly doesn't seem to work😒

**Comment by yoshinorin** [comment, marked as answer]
@forever218 

1. copy the folders (source, theme, etc...)
2. run `npm install` in your project directory
3. run `npm install` in your `theme` directory (if needed)

**Reply by forever218** [reply]
thanks a lots
